src/git-version.cr

```diff
@@ -0,0 +1,141 @@
+require "file_utils"
+
+require "semantic_version"
```

Comment:
where is this coming from? I might be missing something here. This seems to work but I can't find this or the definition of the `SemanticVersion` anywhere.